tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/probe/PublicAssetProbe.scala

```diff
@@ -0,0 +1,21 @@
+package com.linagora.tmail.james.common.probe
+
+import com.google.inject.AbstractModule
+import com.google.inject.multibindings.Multibinder
+import com.linagora.tmail.james.jmap.publicAsset.{PublicAssetCreationRequest, PublicAssetRepository, PublicAssetStorage}
+import jakarta.inject.Inject
+import org.apache.james.core.Username
+import org.apache.james.utils.GuiceProbe
+import reactor.core.scala.publisher.SMono
+
+class PublicAssetProbeModule extends AbstractModule {
+  override def configure(): Unit =
+    Multibinder.newSetBinder(binder(), classOf[GuiceProbe])
+      .addBinding()
+      .to(classOf[PublicAssetProbe])
+}
+
+class PublicAssetProbe @Inject()(publicAssetRepository: PublicAssetRepository) extends GuiceProbe {
```

Comment:
We don't need to rely on the NO REVIEW `/get method` commit to get the `PublicAssetProbe` IMO (can duplicate a bit and resolve later).